exercises/difference-of-squares/HINTS.md

```diff
@@ -1,3 +1,4 @@
 ## Hints
 For this exercise the following F# feature comes in handy:
```

Comment:
Sounds good, thanks @ErikSchierboom. I didn't know what our merging policy was, but I'll stick to "squash and merge" in the future!